torch/nn/modules/module.py

```diff
@@ -17,6 +17,15 @@ def __repr__(self):
     __str__ = __repr__
 
 
+class ModuleAttributeError(AttributeError):
+    """ When `__getattr__` raises AttributeError inside a property,
+    AttributeError is raised with the property name instead of the
+    attribute that initially raised AttributeError, making the error
+    message uninformative. Using `ModuleAttributeError` instead
+    fixes this issue."""
+ModuleAttributeError.__module__ = 'torch'
```

Comment:
Err, how come we do this? Is this really available as `torch.ModuleAttributeError`?

Reply:
No, it's not available as `torch.ModuleAttributeError` but as `torch.nn.modules.module.ModuleAttributeError`. 
Actually `numpy` did like this but it made more sense in their case, given where it was defined. 
I'm removing it.